CMakeLists.txt

```diff
@@ -474,6 +482,25 @@ if(WITH_CORE)
   find_package(EXIV2 REQUIRED)
 
   # optional
+  if (WITH_SFCGAL)
+
+    if(POLICY CMP0167) # avoid deprecation warning from CGAL
+      cmake_policy(SET CMP0167 NEW)
+    endif()
+
+    find_package(SFCGAL) # SFCGAL provider
+    message(STATUS "Found SFCGAL: ${SFCGAL_VERSION} ${SFCGAL_DIR}")
+    if(SFCGAL_VERSION VERSION_LESS "2.1")
```

Comment:
2.1 is a very aggressive minimum version. It's not even available in fedora 42. Could this be dropped to a lower version?

Reply:
`find_package` supports a minimum version argument, better use that